### ISSUE TITLE
All Pokemon Timer show up as < 5:30

### ISSUE THREAD

**Issue by Scabbers352** [AHAAAAAAA/PokemonGo-Map#1680]
Aside from all Pokemon not showing up on the screen when using -t 1, I am also noticing that all my timers for time remaining are showing up < 5:30. No Pokemon have a time remaining > 5:30.

I have tested this on both MAC and PC.

**Comment by shivampaw** [comment]
i have timers above 5:30

What branch are you using?

**Comment by Scabbers352** [comment]
Most recent one, master branch. It's been doing this since I downloaded the updated master this morning. I was using the initial branch before, the one with example.py. That one still works as intended and the timers are correct also.

**Comment by shivampaw** [comment]
Can you try the develop branch?

**Comment by Scabbers352** [comment]
I downloaded the dev branch and installed and tried to think about the issue more.. I noticed my computer clock was not set to automatic and in fact it looked like it was about 9 minutes ahead of the server. I guess the original example.py got the time from the server? and now the code was changed to use the time from the computer. So aside from the example.py and runserver looking a little different I got the time issue fixed. Thanks!
